src/OnboardingSPA/components/SkipButton/index.js

```diff
@@ -48,6 +49,7 @@ const SkipButton = ( { callback = false } ) => {
 			}
 			setFlow( currentData );
 		}
+		await HiiveAnalytics.dispatchEvents();
```

Comment:
Why do we have this here? don't we dispatch on page change?

Reply:
Moved it to inside the `basic-info` check to handle the reload